src/test/resources/io/managed/services/test/quickstarts/quickstarts.feature

```diff
@@ -25,6 +25,37 @@ Feature: Quick starts for OpenShift Streams for Apache Kafka
     When you create the Kafka topic test-topic
     Then the Kafka topic test-topic is listed in the topics table
 
+  Scenario: Java test
+
+    Given you have a Red Hat account
+    * you are logged in to the OpenShift Streams for Apache Kafka web console
```

Comment:
we will be changing this method a bit, I am not sure if this will not bring a conflict with your PR later on (99% it should not). So once  https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/pull/449 (today) will be merged we can rebase and merge.